clvm_tools/cmds.py

```diff
@@ -97,8 +98,13 @@ def as_bin(streamer_f):
 
 
 def run(args=sys.argv):
-    return launch_tool(args, "run", default_stage=2)
-
+    parser = argparse.ArgumentParser(add_help=False)
```

Comment:
Does `clvm_tools_rs` also have a `run` and `brun` tool?

Reply:
Should removing dependency on `clvm_tools` completely be a goal? The only nit I see in doing that is the curry stuff, which could be easily moved to `chia-blockchain` wallet utility code (near where `Program` is, for example).

Reply:
i never implemented stage 1 in clvm_tools_rs since it wasn't being used, so that's the reason for the stage dance here.  since the new features are all in the 'run' personality i made this change conservatively to preserve compatibility.  will remove stage 1 and deprecate it though and remove this.